openmdao/components/test/test_meta_model.py

```diff
@@ -352,6 +352,11 @@ def test_derivatives(self):
         prob['x'] = 0.125
         prob.run()
 
+        Jf = prob.calc_gradient(['x'], ['meta.f'], mode='fwd')[0][0]
+        Jr = prob.calc_gradient(['x'], ['meta.f'], mode='rev')[0]
+
+        assert_rel_error(self, Jf-Jr, 0.0, 1.0e-8)
```

Comment:
I'm not a fan of testing this way, since it doesn't mean that the values are correct even if Jf-Jr == 0.  It just means that their values are the same.